deepspeed/runtime/swap_tensor/utils.py

```diff
@@ -14,6 +14,7 @@
 
 MIN_AIO_BYTES = 1024**2
 AIO_ALIGNED_BYTES = 1024
+MIN_SWAPPABLE_BYTES = MIN_AIO_BYTES
```

Comment:
is this `1024**2` the minimum size to achieve good swap tput for All/Most generations of PCIe?

Reply:
Yes, 1M is a decent size for swap performance.